kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/manager/ResourceManager.java

```diff
@@ -158,6 +200,58 @@ public final <T extends HasMetadata> void deleteResource(T... resources) {
         }
     }
 
+    public void deleteResources() {
+        if (!storedResources.containsKey(getTestContext().getDisplayName()) || storedResources.get(getTestContext().getDisplayName()).isEmpty()) {
+            LOGGER.info("In context {} is everything deleted", getTestContext().getDisplayName());
+        }
+        else {
+            LOGGER.info("Deleting all resources for {}", getTestContext().getDisplayName());
+        }
+
+        // if stack is created for specific test suite or test case
+        AtomicInteger numberOfResources = storedResources.get(getTestContext().getDisplayName()) != null
+                ? new AtomicInteger(storedResources.get(getTestContext().getDisplayName()).size())
+                :
+                // stack has no elements
+                new AtomicInteger(0);
+        while (storedResources.containsKey(getTestContext().getDisplayName()) && numberOfResources.get() > 0) {
+            Stack<ResourceItem<?>> stack = storedResources.get(getTestContext().getDisplayName());
+
+            while (!stack.isEmpty()) {
+                ResourceItem<?> resourceItem = stack.pop();
+
+                try {
+                    resourceItem.throwableRunner().run();
+                }
+                catch (Exception e) {
+                    LOGGER.atTrace().log(Arrays.toString(e.getStackTrace()));
+                }
+                numberOfResources.decrementAndGet();
+            }
+        }
+        storedResources.remove(getTestContext().getDisplayName());
+    }
+
+    /**
+     * Synchronizing all resources which are inside specific extension context.
+     * @param <T> type of the resource which inherits from HasMetadata f.e Kafka, KafkaConnect, Pod, Deployment etc..
+     */
+    @SuppressWarnings(value = "unchecked")
+    public final <T extends HasMetadata> void synchronizeResources() {
```

Comment:
is this name accurate? It looks like it's waiting for all stored resources for this test to be ready.

Reply:
Removed as it was unused after some removals